tests/tests/test_distance/test_distance.py

```diff
@@ -0,0 +1,39 @@
+"""
+"""
+import phoebe
+
+
+def test_distance_scaling_alt_backend(verbose=False, plot=False):
+    b = phoebe.Bundle.default_binary()
```

Comment:
is there a reason you prefer a call to Bundle even though default_binary() is exposed in top-level?

Reply:
no, I just copy-pasted from another test that was probably written before the shortcut.  They're identical though.

Reply:
it would be nice to have a consistent treatment throughout the test suite, so my suggestion would be to decide and stick with one, especially in the newly written code.

Reply:
I've done most of that work already when I ensured pytest compatibility, so I don't think it'll be nearly as daunting to finish. But at least for newly added tests let's try to be consistent?